0001/README.md

```diff
@@ -253,18 +254,18 @@ We separate *client authentication* and *endpoint identification*.
 During client authentication, client and server should verify each other: client verifies server identity and server verifies that client is authorized to access server.
 Note that multiple clients MAY share the same credentials, in which case server can not identify clients: it merely verifies that client knows a shared secret.
 
-Client authentication and *session-wide encryption* are usually handled simultaneously by separate encapsulating protocols (TLS, DTLS, IPSec, none, etc.), so it is not described in this RFC.
-However, a possible authentication method is using MQTT CONNECT packet with name and password.
-It is also possible to disable client authentication at all, but that's insecure and is not recommended.
+Client authentication and *session-wide encryption* are usually handled simultaneously by separate encapsulating protocols (TLS, DTLS, IPSec, none, etc.), and are therefore not described in this RFC.
+However, a possible authentication method would use an MQTT CONNECT packet with name and password.
+You can also completely disable client authentication, but that's insecure and is not recommended.
```

Comment:
RFCs should avoid being personal.

Reply:
I'm not completely sure about this. While 'you can' doesn't seem very personal to me as it is merely the same as 'one can', [RFCs](https://www.rfc-editor.org/rfc-index.html) are quite personal all over the place, the use of personal pronouns seems completely unrestricted there. We need to decide whether to follow the original RFC style on this matter or make our own amendment. If we decide to go with the non-personal approach, I'll need to our other RFC's, too.